src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp

```diff
@@ -0,0 +1,149 @@
+    std::wstring m_query =
+        L"SELECT Origin FROM kibana_sample_data_flights";
+};
+
+TEST_F(TestPagination, Fetch15Rows) {
+    int total_rows = 15;
+    std::wstring fetch_size_15_conn_string =
+        use_ssl ? L"Driver={Elasticsearch ODBC};"
+                  L"host=https://localhost;port=9200;"
+                  L"user=admin;password=admin;auth=BASIC;useSSL="
+                  L"1;hostnameVerification=0;logLevel=0;logOutput=C:\\;"
+                  L"responseTimeout=1;fetchSize=15"
+                : L"Driver={Elasticsearch ODBC};"
+                  L"host=localhost;port=9200;"
+                  L"user=admin;password=admin;auth=BASIC;useSSL="
+                  L"0;hostnameVerification=0;logLevel=0;logOutput=C:\\;"
+                  L"responseTimeout=1;fetchSize=15";
+    ASSERT_EQ(SQL_SUCCESS,
+              SQLDriverConnect(
+                  m_conn, NULL, (SQLTCHAR*)fetch_size_15_conn_string.c_str(),
+                  SQL_NTS, m_out_conn_string, IT_SIZEOF(m_out_conn_string),
+                  &m_out_conn_string_length, SQL_DRIVER_PROMPT));
+    EXPECT_EQ(total_rows, GetTotalRowsAfterQueryExecution());
+}
+
+TEST_F(TestPagination, NoFetchSize) {
+    // Default size when pagination is disabled i.e. fetch size is 0.
+    int total_rows = 200;
+    std::wstring fetch_size_15_conn_string =
+        use_ssl ? L"Driver={Elasticsearch ODBC};"
+                  L"host=https://localhost;port=9200;"
+                  L"user=admin;password=admin;auth=BASIC;useSSL="
+                  L"1;hostnameVerification=0;logLevel=0;logOutput=C:\\;"
+                  L"responseTimeout=1;"
+                : L"Driver={Elasticsearch ODBC};"
+                  L"host=localhost;port=9200;"
+                  L"user=admin;password=admin;auth=BASIC;useSSL="
+                  L"0;hostnameVerification=0;logLevel=0;logOutput=C:\\;"
+                  L"responseTimeout=1;";
+    ASSERT_EQ(SQL_SUCCESS,
+              SQLDriverConnect(
+                  m_conn, NULL, (SQLTCHAR*)fetch_size_15_conn_string.c_str(),
+                  SQL_NTS, m_out_conn_string, IT_SIZEOF(m_out_conn_string),
+                  &m_out_conn_string_length, SQL_DRIVER_PROMPT));
+    EXPECT_EQ(total_rows, GetTotalRowsAfterQueryExecution());
+}
+
+int main(int argc, char** argv) {
+#ifdef __APPLE__
+    // Enable malloc logging for detecting memory leaks.
+    system("export MallocStackLogging=1");
+#endif
+    testing::internal::CaptureStdout();
+    ::testing::InitGoogleTest(&argc, argv);
+
+    int failures = RUN_ALL_TESTS();
+
+    std::string output = testing::internal::GetCapturedStdout();
+    std::cout << output << std::endl;
+    std::cout << (failures ? "Not all tests passed." : "All tests passed")
+              << std::endl;
+    WriteFileIfSpecified(argv, argv + argc, "-fout", output);
+
+#ifdef __APPLE__
+    // Disable malloc logging and report memory leaks
+    system("unset MallocStackLogging");
+    system("leaks itodbc_info > leaks_itodbc_info");
```

Comment:
Update filename here
```suggestion
    system("leaks <executable-name> > leaks_<executable-name>");
```

Reply:
done